nrm_django/Procfile

```diff
@@ -0,0 +1 @@
+web: python manage.py runserver 0.0.0.0:8080
```

Comment:
I know this isn't ideal, but it seems adequate for small-group prototyping. I'll log an issue to implement a more robust server.

Reply:
Ticket number here for breadcrumbs when you make it?

Reply:
Good idea. #78 is logged for it.